app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,70 @@
+require 'date'
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all
+  end
+
+  def edit
+    @task = Task.find(params[:id])
+  end
+
+  def update
+    # task_updates = params[:task]
+    # @task = Task.find(params[:id])
+    # @task.update_attributes(task_updates)
+    # @task.save
+
+    @task = Task.find(params[:id])
+    @task.name = params[:task][:name]
+    @task.description = params[:task][:description]
+    @task.completion_date = params[:task][:completion_date]
+    task_params
```

Comment:
Calling `task_params` here does not actually get us anything, because the method itself doesn't change any variables or update the database.

Instead the way that we would use the strong params code in `task_params` is to do a short-hand version of this whole `update` action:
```ruby
def update
  task = Task.find(params[:id])
  task.update(task_params)
  redirect_to task_path(task)
end
```